doc/roles_collections.rst

```diff
@@ -0,0 +1,544 @@
+
+.. code-block:: Yaml
+
+  ---
+
+  - name: Create SSH authorized_keys directory
+    ansible.builtin.file:
+      path: /etc/ssh/authorized_keys
+      state: directory
+      owner: root
+      group: root
+      mode: "0755"
+      seuser: system_u
+      serole: object_r
+      setype: sshd_key_t
+      selevel: s0
+
+  - name: Configure SSHd
+    ansible.builtin.template:
+      src: etc/ssh/sshd_config.j2
+      dest: "{{ ssh_daemon_cfg }}"
+      owner: root
+      group: root
+      mode: "0644"
+      seuser: system_u
+      serole: object_r
+      setype: etc_t
+      selevel: s0
+      validate: "{{ ssh_daemon_bin }} -t -f %s"
+    notify:
+      - Restart sshd
+
+If necessary, you can add additional tags to individual tasks inside
+the imported files.  However, since this ad-hoc tag list overrides the
+one defined in ``main.yml``, you must also provide all the tags from
+``main.yml`` again for the single task:
+
+Good example:
+
+.. code-block:: Yaml
+
+  - name: Install SSH related packages
+    ansible.builtin.package:
+      name: "{{ ssh_packages }}"
+      state: present
+    tags:
+      # This tag is added only for this task
+      - "role::sshd:packages"
+      # These two tags must  be provided again, as the tag list from main.yml is overwritten by this tag list.
+      - "role::sshd"
+      - "role::sshd:install"
+
+Bad example:
+
+.. code-block:: Yaml
+
+  - name: Install SSH related packages
+    ansible.builtin.package:
+      name: "{{ ssh_packages }}"
+      state: present
+    tags:
+      - "role::sshd:packages"
+
+This task is no longer executed when run via ``--tags role::sshd``.
+
+Variables
+---------
+
+Variables in ``vars/`` are used for static data, e.g. package-, service-
+and filenames.  Only use ``vars/`` for data that does not change on a
+host-by-host basis, for that use the defaults!
+
+The variables stored in ``vars/`` can be loaded dynamically.  This can
+be used to e.g. load OS-dependent variables.  The example above uses
+this to load the ``ssh_packages`` variable dependent on the
+``os_family`` host fact.
+
+To achieve this, you put the variables into files named after ``os_family`` inside the ``vars/`` directory:
+
+* ``Debian.yml``
+* ``RedHat.yml``
+
+If there are special variables for some operating systems, you can specify
+those in the files named:
+
+* ``Debian_11.yml``
+* ``Debian_12.yml``
+* ``CentOS_7.yml``
+* ``CentOS_8.yml``
+* ``CentOS_9.yml``
+* ``Ubuntu_20.yml``
+* ``Ubuntu_22.yml``
+* ...
+
+This logic is implemented using the ``with_first_found`` iterator in
+the example above.  For more information, check out the documentation
+on `Loops <https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html>`_.
+  
+By our convention, each variable name start with ``<rolename>_`` and
+the name contains only lower case letters, numbers and underline
+``_``:
+
+.. code-block:: Yaml
+
+  ---
+
+  # ssh related packages
+  ssh_packages:
```

Comment:
```suggestion
  sshd_packages:
```
The role tags are sshd::install so following that logic the variables should also start with sshd_